src/cli/commands/licenses.js

```diff
@@ -101,65 +99,4 @@ export const {run, setFlags, examples} = buildSubCommands('licenses', {
   async list(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
     await list(config, reporter, flags, args);
   },
-
-  async generateDisclaimer(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
```

Comment:
Well, since no tests cover this, I think I'll add a test case for this since it was very confusing to me not finding a single call to this function.